### PR TITLE
[Form] Full form example & header classes

### PR DESCRIPTION
**Pull request by jeremie-lucca**
## Description

Full form example & header classes

-----

image

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2563/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/13166/ui-diff)

File: stories/documentation/forms/examples/html&css/form-basic.stories.ts [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+export default {
+	title: 'Documentation/Forms/Examples/HTML&CSS',
+	argTypes: {
+	},
+} as Meta;
+
+function getTemplate(args: FieldsetBasicStory): string {
+	return `<form class="form">
+	<header class="form-header">
```
Comment (vvalentin-lucca) [inline stories/documentation/forms/examples/html&css/form-basic.stories.ts:14]: I hadn't thought about it yesterday, but to put a `header`, you have to make sure there's sectioning content above it. `form` isn't one of them.

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2563/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/13214/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2563/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/14106/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2563/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/17304/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2563/storybook

**Review by vvalentin-lucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2563/storybook

**Pull request merged**